bedrock/etc/templates/etc/firefox/retention/thank-you-a.html

```diff
@@ -47,7 +47,7 @@
     <article>
       <h1>{{ _('Get Firefox on your phone') }}</h1>
       <div class="time">{{ _('2 min install') }}</div>
-      <a href="https://support.mozilla.org/kb/install-firefox-android-device-using-google-play">{{ _('Download Firefox') }}</a>
+      <a href="https://www.mozilla.org/firefox/mobile-download/desktop/">{{ _('Download Firefox') }}</a>
```

Comment:
Should use our `url` helper here and below: `{{ url('firefox.mobile-download-desktop') }}`

Reply:
Oh interesting. 1. I didn't even notice that link was in moz.org but also 2. that's a cool helper!

Reply:
Yeah, keeps all links pointing to the same environment, which is pretty critical for not going crazy. 😉

Reply:
I found another link I used on moz.org too, so updating that too.